index.adoc

```diff
@@ -1,178 +1,97 @@
-
-|===
-|Tools to build and deploy WildFly on Kubernetes & OpenShift
-
-|link:wildfly-charts[Helm Chart for WildFly]
-|link:wildfly-operator[Kubernetes Operator for WildFly]
-|link:wildfly-s2i[WildFly S2I (Source to Image) builder and runtime images]
-|link:wildfly-container[WildFly container images]
-
-|===
-
-== Previous WildFly Releases
-
-|===
-|WildFly Release | Jakarta EE Version | Java EE Version
-
-|link:34[34.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
-
-|link:33[33.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
-
-|link:32[32.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
-
-|link:31[31.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
-
-|link:30[30.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+== Build & Deploy WildFly on the Cloud
 
-|link:29[29.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+There are different tools to build and deploy WildFly on the Cloud, 
 
-|link:28[28.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+=== WildFly Container
 
-|link:27[27.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+Our original container image that provides a standalone WildFly server. 
+Use this image to add your deployments and build your application image.
 
-|link:26.1[26.1.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Learn more about link:wildfly-container[WildFly Container].
 
-|link:26[26.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+=== Helm Chart
 
-|link:25[25.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Build & Deploy your Java applications with WildFly on Kubernetes and OpenShift with a Helm Chart.
 
-|link:24[24.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Learn more about the link:wildfly-charts[Helm Chart for WildFly].
 
-|link:23[23.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9/apidocs/[EE 9] Preview)
-|
+=== Operator for Kubernetes & OpenShift
 
-|link:22[22.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9/apidocs/[EE 9] Preview)
-|https://javaee.github.io/javaee-spec/javadocs[Java EE 8]
+The WildFly Operator simplifies deployment and maintenance of WildFly containerized applications
```

Comment:
Maybe "deployment and maintenance of stateful WildFly containerized applications" ? Maybe it sounds a bit more appropriate in that way.